### DISCUSSION TITLE
Review

### DISCUSSION THREAD

**Discussion by hadoan149** [lab-brussels-1/HYF-redesign-group-2#16] [Q&A] [answered]
Got things need reviewing? Feel free to put them here

**Comment by sammou00** [comment, marked as answer]
Hello,
Wow very nice user-personas.

![image](https://user-images.githubusercontent.com/67711948/163689412-6d969bcc-ab3f-4b51-8a22-e75a5fba6bb1.png)
this hidden file should be added to `.gitignore`

- using  short and descriptive comments is very useful in programming not just html, CSS and JavaScript but in all programming languages.
